AU2/database/model/Assassin.py

```diff
@@ -24,6 +24,9 @@ class Assassin(PersistentFile):
     college: str
     notes: str
     is_police: bool
+    # we cannot delete assassins because that will break thingsm,

```

Comment:
```suggestion
    # we cannot delete assassins because that will break things,
```

Some examples of things maybe? (e.g., events which take in assassin identifiers)